common.sh

```diff
@@ -427,6 +428,18 @@ print_home_dirs() {
   conf_get_section "$_uconf_file" "home-dirs"
 }
 
+#Function adds a package to special_pkg_list
+add_pkg_to_kickstart() {
+  [ $# -eq 0  ] && {
+    echo "Missing parameters! Any package will be added." >&2
```

Comment:
May log_debug will be better in this case. Not sure about that. What do You think?
```bash
log_debug "Missing parameters! Any package will be added."
```